build.properties

```diff
@@ -4,17 +4,17 @@ mod_id=passableleaves
 mod_name=Passable Leaves
 mod_desc=Minecraft core mod that makes leaves passable.
 mod_version=4.1.0.0
-mc_version=1.10.2
+mc_version=1.11.2
 mod_url=https://github.com/Team-RTG/PassableLeaves
 mod_author="Team RTG"
 mod_creds="Based on code donated by HellFirePvP for the Appalachia mod."
 mod_logo=assets/passableleaves/logo.png
-mcf_version=12.18.2.2099
-mcf_minver=12.18.1.2011
+mcf_version=13.20.0.2228
```

Comment:
This mod is extremely limited in what it does, but it should probably be built against the latest Forge to ensure that it does work with it.